nailgun/entities.py

```diff
@@ -3819,6 +3819,34 @@ def search_normalize(self, results):
         return super(Interface, self).search_normalize(results)
 
 
+class JobTemplate(
+        Entity,
+        EntityReadMixin,
+        EntitySearchMixin):
+    """A representation of a Provisioning Template entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
```

Comment:
I think there are some fields missing like template inputs, audit comment, effective user, and description format.

Reply:
@manuelbonk , Actually I have created that template just for the view job template purpose for Foreman Templates future. So the minimal params are populated. I can remove the entire entity in favor of ur PR https://github.com/SatelliteQE/nailgun/pull/501.